api/v1/users/PKPUserHandler.inc.php

```diff
@@ -199,10 +199,10 @@ public function getReviewers($slimRequest, $response, $args)
             'reviewStage',
             'searchPhrase',
             'status',
+            'showOnlyReviewersFromPreviousRoundsSubmissionId',
```

Comment:
Let's call this `assignedToReviewRoundIds` here and throughout. And instead of a `boolean`, it should accept an `array` of review round ids. 

This will make it more consistent with other user query params (eg - `assignedToStage`, etc). And the API will be more flexible so that we can use the same param for other things (ie - we could replace [this code](https://github.com/pkp/pkp-lib/blob/main/controllers/grid/users/reviewer/form/AdvancedSearchReviewerForm.inc.php#L78-L85) to get all reviewers assigned in the current round).

Reply:
in our case, we don't want only the reviewers assigned to a round but those who have completed their reviews.

I wonder if we shouldn't rather pass to the PKP\components\listPanels\PKPSelectReviewerListPanel params, as for currentlyAssigned, a table with the reviewerId (previouslyAssigned) of completed reviews. And in the UI, we filter on these reviewerIds without making an additional query.

Reply:
> then we can always show those reviewers at the top of the list with a notice that says "This reviewer completed a review in the last round."

Do you mean that this is not a filter, we always show previous reviewers ? and activate a filter don't change this static list ?
If this is the case, these Reviewers should be exclude from the all reviewers list.

To be honest, it seems more complicated than pass a array of reviewerIds.
I have updated my PR with this:
- pass to PKP\components\listPanels\PKPSelectReviewerListPanel an array of reviewers Id from the last round (and with completed reviews) named lastRoundReviewerIds
- use this as a value for a pkp-filter to pass it to API
- use the builtin filterByUserIds of the user API to filter reviewers

Can you check this update ?
Thank you

Reply:
Sorry for the delay in looking into this, @forgive38. I tried to look at it today, but I think it maybe needs a rebase against the `main` branch. I had trouble running the PRs as-is, due to fatal errors that looked like they're related to some big merges we've had recently. Can you rebase and then let me know and I'll take another look?